modules/serverless-rn-1-6-0.adoc

```diff
@@ -0,0 +1,71 @@
+
+[id="serverless-rn-1-6-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.6.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-6-0_{context}"]
+== New features
+* {ServerlessProductName} 1.6.0 is available on {product-title} 4.3 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.13.1.
+* {ServerlessProductName} has been updated to use Knative `kn` CLI 0.13.1.
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.13.1.
+* The `serving.knative.dev` API group has now been fully deprecated, and is replaced by the `operator.knative.dev` API group.
++
+You must complete the steps described in the {ServerlessProductName} 1.4.0 release notes, related to updating from the `serving.knative.dev` API group to the `operator.knative.dev` API group, before you can upgrade to the latest version of {ServerlessProductName}.
++
+[IMPORTANT]
+====
+This change causes unspecified commands, such as `oc get knativeserving`, to become unreliable and not always work correctly.
+
+After you have completed the upgrade to {ServerlessProductName} 1.6.0, you will need to remove the old CRD to fix this issue.
+You can remove the old CRD by entering the following command:
+----
+$ oc delete crd knativeservings.serving.knative.dev
+----
+====
+* The *Subscription Update Channel* for new {ServerlessProductName} releases has been updated from `techpreview` to `preview-4.3`.
++
+[IMPORTANT]
+====
+You must update your channel using the upgrade documentation to consume the latest {ServerlessProductName} version.
+====
+* {ServerlessProductName} now supports the use of `HTTP_PROXY`.
+* {ServerlessProductName} now supports `HTTPS_PROXY` cluster-proxy settings.
++
+[NOTE]
+====
+This does not include custom certificates.
+====
+* The `KnativeServing` CRD is now hidden from the Developer Catalog by default, so that only users with cluster administrator permissions can view it.
+* Parts of the `KnativeServing` control plane and data plane are now deployed as highly available (HA) by default.
+* Kourier is now actively watched and reconciles changes automatically.
+* {ServerlessProductName} now supports use on {product-title} nightly builds.
+
+[id="fixed-issues-1-6-0_{context}"]
+== Fixed issues
+* In previous versions, the `oc explain` command did not work correctly. The structural schema of the `KnativeServing` CRD has been updated in {ServerlessProductName} 1.6.0 so that the `oc explain` command now works correctly.
+* In previous versions, it was possible to create more than one `KnativeServing` CR. Multiple `KnativeServing` CRs are now prevented synchronously in {ServerlessProductName} 1.6.0. Attempting to create more than one `KnativeServing` CR now results in an error.
+* In previous versions, {ServerlessProductName} was not compatible with {product-title} deployments on GCP. This issue has been fixed in {ServerlessProductName} 1.6.0.
```

Comment:
Please expand the first use of GCP.

Reply:
I'm not sure what you mean by this @kalexand-rh ?

Reply:
I'd say "Google Cloud Platform"